System.Reflection.Emit.Experimental.AssemblyBuilderSave.sln

```diff
@@ -3,9 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.26124.0
 MinimumVisualStudioVersion = 15.0.26124.0
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Experiment", "src\Experiment\src\Experiment.csproj", "{B7977360-6671-4707-9A1C-1C29D5BE2674}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Reflection.Emit.Experimental.AssemblyBuilderSave.csproj", "src\Experiment\src\System.Reflection.Emit.Experimental.AssemblyBuilderSave.csproj", "{B7977360-6671-4707-9A1C-1C29D5BE2674}"
```

Comment:
I would remove `AssemblyBuilderSave` from the assembly name and the namespace. Call it just `System.Reflection.Emit.Experimental`.

I expect that you will add `AssemblyBuilder` type with `Save` method in this namespace. So if you run with `System.Reflection.Emit.Experimental.AssemblyBuilderSave` as the namespace, you would end up with `System.Reflection.Emit.Experimental.AssemblyBuilderSave.AssemblyBuilder.Save` that is very long and repetitive.